docs/standard/class-library-overview.md

```diff
@@ -108,9 +109,9 @@ ass objects|<xref:System.Object>|The root of the object hierarchy.|**Object**|**
   
  In addition to the base data types, the <xref:System> namespace contains over 100 classes, ranging from classes that handle exceptions to classes that deal with core runtime concepts, such as application domains and the garbage collector. The <xref:System> namespace also contains many second-level namespaces.  
   
- For more information about namespaces, browse the [.NET Framework Class Library](http://go.microsoft.com/fwlink/?LinkID=227195). The reference documentation provides a brief overview of each namespace as well as a formal description of each type and its members.  
+ For more information about namespaces, use the [.NET API Browser](https://docs.microsoft.com/dotnet/api) to browse the .NET Class Library. The API reference documentation provides documentation on each namespace, its types, and each of their members.  
   
 ## See Also  
  [Common Type System](../../docs/standard/base-types/common-type-system.md)  
- [.NET Framework Class Library](http://go.microsoft.com/fwlink/?LinkID=227195)  
+ [.NET API Browser](https://docs.microsoft.com/dotnet/api)  
```

Comment:
@rpetrusha should we use a relative link to `../../api/index.md` for .NET API Browser? (The same remark refers to the paragraph above.)

Reply:
The page there appears to be a sub, so I'm not sure that it will render properly, @pkulikov. @mairaw, do you know? Should the link be changed to a relative link in our repo, or should it be left as a link to docs?

Reply:
@rpetrusha ah, OK. I've found the third way that is used currently in the docs. The [raw file](https://raw.githubusercontent.com/dotnet/docs/c6341e749d70800a701288d2dcd2605c7cf4e0f2/docs/framework/get-started/index.md) of **See Also** section of [this page](https://docs.microsoft.com/en-us/dotnet/framework/get-started/) contains the line `[.NET API Browser](/dotnet/api/)`

Reply:
Given that we'll split the repo soon, I think we should keep it as you did.

Reply:
@mairaw could you please tell more about the split of the repo, or that's too early yet?

Reply:
@pkulikov We'll post more details once it's done but we're working to split the conceptual and api docs into two different repos and we'll have a third one for the samples. This should reduce build times, especially for the conceptual docs.

Reply:
@mairaw that's clear, thank you.